package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "opg-sirius-lpa-frontend",
+  "version": "0.1.0",
+  "license": "MIT",
+  "scripts": {
+    "build": "webpack",
```

Comment:
Might be worth trying to switch to esbuild some time to bring our dependencies under control, if anyone fancies trying it.

Reply:
Sounds good, but I might just add a spike for it